2021/25xxx/CVE-2021-25217.json

```diff
@@ -1,18 +1,124 @@
 {
```

Comment:
'value' is a required property

Reply:
Oops.  I'll rework and submit a new MR.

Reply:
I've removed the extraneous empty problemtype_data/description